src/server/JasmineGraphInstanceService.cpp

```diff
@@ -1378,18 +1390,447 @@ void *instanceservicesession(void *dummyPt) {
                 fileName = utils.trim_copy(fileName, " \f\n\r\t\v");
                 instance_logger.log("Received File name: " + fileName, "info");
 
-                string aggregateLocation = utils.getJasmineGraphProperty("org.jasminegraph.server.instance.aggregatefolder");
+                string aggregateLocation = utils.getJasmineGraphProperty(
+                        "org.jasminegraph.server.instance.aggregatefolder");
                 string fullFilePath = aggregateLocation + "/" + fileName;
                 string result = "false";
 
-                bool fileExists = utils.fileExists(fullFilePath);
+                bool fileExists = utils.fileExists(fullFilePath);
+
+                if (fileExists) {
+                    result = "true";
+                }
+
+                write(connFd, result.c_str(), result.size());
+                instance_logger.log("Sent : " + result, "info");
+            }
+        } else if (line.compare(JasmineGraphInstanceProtocol::CREATE_BLOOM_FILTERS) == 0) {
+            instance_logger.log("Received : " + JasmineGraphInstanceProtocol::CREATE_BLOOM_FILTERS, "info");
+            write(connFd, JasmineGraphInstanceProtocol::OK.c_str(), JasmineGraphInstanceProtocol::OK.size());
+            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::OK, "info");
+            bzero(data, INSTANCE_DATA_LENGTH);
+            read(connFd, data, INSTANCE_DATA_LENGTH);
+            string graphID = (data);
+            graphID = utils.trim_copy(graphID, " \f\n\r\t\v");
+            instance_logger.log("Received Graph ID: " + graphID, "info");
+            write(connFd, JasmineGraphInstanceProtocol::SEND_PARTITION_ID.c_str(),
+                  JasmineGraphInstanceProtocol::SEND_PARTITION_ID.size());
+            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::SEND_PARTITION_ID, "info");
+            bzero(data, INSTANCE_DATA_LENGTH);
+            read(connFd, data, INSTANCE_DATA_LENGTH);
+            string partitionID = (data);
+            instance_logger.log("Received partition ID: " + partitionID, "info");
+
+            //pthread_mutex_lock(&file_lock);
```

Comment:
We can remove the commented lines isn't it?

Reply:
This code is not a part of my changes, should I make changes here?